src/stores/b2b-purchase-order-flow.md

```diff
@@ -0,0 +1,83 @@
+---
+b2b_only: true
+title: Purchase order workflow
+group: getting-started
+---
+
+Purchase orders allow companies to track and control spending. When purchase orders are enabled for a company account, all orders are automatically processed as purchase orders.
+
+The purchase order workflow can vary in a few ways:
+
+- If no approval rules are set, purchase orders can be placed and the order completed directly.
+- If approval rules are established, regular users will go through the approval process.
+- Company admins can bypass the approval rules.
```

Comment:
We are not using the shortened `admin` here because it is confusing with our other usage of the term.

```suggestion
- Company administrators can bypass the approval rules.
```